content/v2.0/query-data/common-queries/median.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Query median values
+seotitle: Query median values in Flux
+list_title: Median
+description: >
+  Use the `median()` function to return all values within the `0.5` quantile
+  (50th percentile) or the median value of input data.
+weight: 202
+menu:
+  v2_0:
+    parent: Common queries
+    name: Median
+v2.0/tags: [query, median]
+related:
+  - /v2.0/query-data/common-queries/percentile-quantile/
+---
+
+Use the [`median()` function](/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/median/)
+to return all values within the `0.5` quantile (50th percentile) or the median value of input data.
+
+## Median calculation methods
+Select from the following methods for calculating the median:
+
+- [estimate_tdigest](#estimate-tdigest)
+- [exact_mean](#exact-mean)
+- [exact_selector](#exact-selector)
+
+### estimate_tdigest
+**(Default)** An aggregate method that uses a [t-digest data structure](https://github.com/tdunning/t-digest)
+to compute an accurate quantile estimate on large data sources.
+Output tables consist of a single row containing the calculated median.
+
+{{< flex >}}
+{{% flex-content %}}
+**Given the following input table:**
+
+| _time | _value |
+| ----- |:------:|
+| 0001  | 1.0    |
+| 0002  | 1.0    |
+| 0003  | 2.0    |
+| 0004  | 3.0    |
+{{% /flex-content %}}
+{{% flex-content %}}
+**`estimate_tdigest` returns:**
+
+| _value |
+|:------:|
+| 1.5    |
+{{% /flex-content %}}
+{{< /flex >}}
+
+### exact_mean
+An aggregate method that takes the average of the two points closest to the quantile value.
+Output tables consist of a single row containing the calculated median.
+
+{{< flex >}}
+{{% flex-content %}}
+**Given the following input table:**
```

Comment:
example that shows a different result, depending on how the median is calculated?